tools/torch_quant/torch_quant/observer.py

```diff
@@ -214,24 +251,30 @@ def forward(self, x):
 
 
 class BiasObserver(Observer):
-    def __init__(self, w_ob: Observer, act_ob: Observer,
-                 dtype: torch.dtype = torch.qint32,
-                 qscheme: torch.qscheme = torch.per_tensor_affine, **kwargs) -> None:
-        super().__init__(**kwargs)
+    def __init__(self, w_ob: Observer, act_ob: Observer, **kwargs) -> None:
+        dtype = torch.qint32
+        is_w_per_channel = is_per_channel(w_ob.qscheme)
```

Comment:
just use `w_ob.per_channel`?

Reply:
Generally, only the qscheme of weight will be per-channel.
Have added activation qscheme to co-determine the qscheme of bias

Reply:
ok, but code can be simplified as `if w_ob.per_channel or act_ob.per_channel:` ?

Reply:
done